data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -15,49 +15,48 @@
 
     [story]
         [part]
-            story= _ "Far to the north of Wesnoth, and hard by the dwarven caves of Knalga, there was once a thriving mixed community known as Dwarven Doors. Food, wood, and cloth from all over Wesnoth and many elvish lands as well were brought here to be sold to the dwarves, while ores, metals and worked dwarven manufactures were bought here to be traded throughout the surface nations. Dwarves and humans worked and lived side by side; even the elves who came to trade mingled more with the other kindred races than was common elsewhere. Dwarven Doors grew populous and wealthy."
+            story= _ "Far to the north of Wesnoth, and close to dwarven caves of Knalga, there was once a thriving mixed community known as Dwarven Doors. Food, wood, and cloth from all over Wesnoth and many elvish lands as well were brought here to be sold to the dwarves, while ores, metals and worked dwarven manufactures were bought here to be traded throughout the surface nations. Dwarves and humans worked and lived side by side; even the elves who came to trade mingled more with the other kindred races than was common elsewhere. Dwarven Doors grew populous and wealthy."
```

Comment:
Sounds like the original was going for an olden style of English, which I understand Wesnoth as a whole goes for. I don't think this is a good change.

Reply:
I agree with wedge. I don't think this is an improvement in this case.